rust/sealevel/environments/testnet4/multisig-ism-message-id/eclipsetestnet/hyperlane/multisig-config.json

```diff
@@ -0,0 +1,117 @@
+{
+    "alfajores": {
+        "threshold": 2,
+        "validators": [
+            "0x2233a5ce12f814bd64c9cdd73410bb8693124d40",
+            "0xba279f965489d90f90490e3c49e860e0b43c2ae6",
+            "0x86485dcec5f7bb8478dd251676372d054dea6653"
+        ],
+        "type": 3
```

Comment:
I created a ticket to address this manual change we needed to do https://github.com/hyperlane-xyz/hyperlane-monorepo/issues/3174